keyboards/ckeys/handwire_101/keymaps/default/config.h

```diff
@@ -0,0 +1,6 @@
+#ifndef CONFIG_USER_H
```

Comment:
`#pragma once`... blah blah blah...

```suggestion
#pragma once
```